cli/api/logs.go

```diff
@@ -49,10 +83,103 @@ type ExportLogsConfig struct {
 //
 // TODO: This code is racy - creating then erasing the output file does not
```

Comment:
Since the output file name is based on a timestamp with resolution of down to the second (RFC3339), the race condition is one where two or more CLI commands that are initiated within the same second on the same host from the same directory.  

Possible solutions:

1. Add some unique salt to the end of the file name (e.g. a random number) Cons - the file names are already >40 chars long.
2. Lower the probability of collision even further by using a timestamp based on milliseconds. Technically, still racy
3. Narrow the time window for potential race further by keeping tgzfile open for the duration of the operation. Move the Close/Remove calls from here https://github.com/control-center/serviced/blob/8d97521b04dbc3e122bf9b993538f3326a044d10/cli/api/logs.go#L204 down to here https://github.com/control-center/serviced/blob/8d97521b04dbc3e122bf9b993538f3326a044d10/cli/api/logs.go#L160. That 
4. Do #2 and #3